### PR TITLE
Add release notes for 0.101

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.101.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ General Changes
 * Add :func:`array_position` function.
 * Add :func:`regexp_split` function.
 * Fix analysis of ``UNION`` queries for tables with hidden columns.
+* Fix precedence of ``JOIN`` when more than one ``JOIN`` exists in a query.
```
Comment (cberner) [inline presto-docs/src/main/sphinx/release/release-0.101.rst]: Give a little more detail on this. Does it effect all queries that have multiple JOINs?

**Comment by haozhun** [comment]
Updated

**Comment by cberner** [comment]
looks good

**Comment by haozhun** [comment]
Merged

**Pull request closed without merging**